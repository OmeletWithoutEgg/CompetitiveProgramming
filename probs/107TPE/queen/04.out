-1439.81 -17135.47
