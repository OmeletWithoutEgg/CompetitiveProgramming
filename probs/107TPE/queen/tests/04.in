1808
-36989 -99040
-31407 -89998
-34108 -85045
-44002 -80448
-41522 -82493
-38075 -79734
-44226 -96150
-43547 -94317
-47342 -98982
-49786 -75659
-51103 -97649
-49457 -86196
-36220 -81620
-49226 -76847
-32980 -92645
-48123 -81405
-37930 -86100
-48547 -80826
-35664 -76237
-31415 -90580
-48464 -85688
-47394 -96073
-49178 -80842
-32591 -80196
-43610 -99440
-50818 -90706
-39575 -93191
-35320 -82185
-36504 -98134
-34961 -77636
-34728 -86572
-40081 -80592
-40166 -97914
-40059 -86754
-50736 -80973
-38323 -80544
-49789 -84672
-40961 -78318
-47273 -96175
-36448 -90653
-35856 -87370
-30633 -96244
-37863 -98309
-49494 -98282
-47188 -79716
-32915 -80624
-37014 -96205
-31134 -87574
-32772 -97397
-46642 -86818
-35464 -81960
-36586 -77294
-49268 -87758
-39462 -92854
-46555 -83860
-38066 -85054
-30741 -83222
-48349 -86241
-46819 -96415
-45758 -80564
-30620 -81849
-50557 -83333
-31411 -95139
-46013 -98002
-34421 -83561
-44128 -75977
-45103 -92036
-35140 -94977
-35851 -84271
-50478 -76399
-49519 -98973
-50907 -87831
-31943 -98862
-43714 -84809
-38136 -89875
-43136 -98592
-33510 -85284
-43923 -85723
-39748 -84042
-33318 -97468
-34800 -90047
-36238 -84481
-39577 -78272
-42681 -78307
-31231 -89881
-31609 -87743
-51136 -95399
-48674 -92360
-36163 -83272
-51041 -75927
-48342 -90415
-41775 -79800
-42738 -99499
-39710 -92672
-35460 -86275
-35696 -96273
-31270 -97780
-33666 -77169
-37955 -86071
-31601 -85589
-38091 -94101
-50207 -97464
-34272 -75729
-43395 -79765
-42292 -96459
-40347 -93375
-32139 -94148
-34965 -99980
-38678 -81589
-47540 -94841
-35108 -81984
-33723 -93894
-32116 -76197
-43275 -99805
-37910 -83559
-48881 -81036
-48091 -97514
-49620 -90850
-42132 -86193
-49187 -98384
-32174 -98106
-33350 -96214
-31752 -90202
-45396 -88245
-43765 -82114
-47777 -89435
-38069 -93080
-51082 -90067
-44767 -79568
-33875 -85811
-45828 -95410
-31097 -88578
-34763 -82217
-51247 -96573
-37199 -82386
-47933 -83796
-48675 -88834
-49545 -84928
-41033 -81821
-34090 -80965
-37810 -92479
-30989 -84037
-34747 -83835
-35392 -92730
-41966 -81991
-31545 -80793
-33998 -92626
-48799 -77816
-50198 -89207
-42125 -82197
-32828 -84986
-49376 -83998
-35164 -87859
-33760 -89026
-31140 -79779
-30976 -100176
-45482 -91909
-31214 -86233
-31185 -98052
-44776 -93367
-32692 -86669
-37498 -94338
-39789 -78289
-42020 -80162
-41919 -95895
-41204 -97732
-35660 -91776
-41716 -100223
-38459 -89491
-48885 -78250
-49691 -79958
-40249 -77927
-47660 -84588
-33741 -86070
-49806 -93973
-48099 -95480
-33526 -97445
-50164 -79197
-41051 -79014
-50378 -76278
-47747 -89250
-31379 -89595
-42811 -95939
-31381 -87688
-43444 -89501
-34890 -76443
-44876 -81764
-42186 -94615
-36052 -94065
-34563 -86077
-42969 -88164
-37673 -91915
-49941 -89290
-39680 -84928
-49979 -84985
-50776 -83587
-42131 -99160
-46808 -81256
-42563 -79092
-36728 -98531
-41080 -98764
-45974 -80798
-40226 -87269
-34341 -75751
-48124 -76553
-50670 -85005
-42235 -90433
-41601 -85941
-33907 -92753
-45539 -96960
-50430 -95414
-43403 -93180
-34199 -90592
-40723 -93825
-47574 -95440
-43174 -88519
-37661 -78109
-45624 -90278
-32774 -88389
-46241 -96586
-39763 -86924
-37523 -80672
-37640 -80160
-48626 -85973
-40168 -85431
-43751 -90819
-45062 -86804
-32710 -96394
-40534 -83792
-30703 -87057
-50764 -85526
-38467 -84087
-37421 -76776
-37858 -80638
-48039 -81383
-42842 -96310
-35410 -86775
-49873 -78120
-47815 -99329
-32132 -84960
-48216 -77088
-35467 -84043
-47142 -87228
-46485 -99420
-33859 -82083
-35641 -76969
-32157 -87309
-47232 -88887
-47062 -76265
-34517 -81260
-41191 -86093
-31881 -97753
-38893 -79330
-45012 -96335
-47516 -81133
-34460 -82041
-50310 -80114
-33115 -86583
-33733 -88110
-49071 -78308
-31680 -87442
-40625 -98828
-41208 -83824
-31791 -98226
-34481 -86133
-50777 -86907
-43436 -97616
-39673 -91734
-49311 -89633
-49127 -99553
-37476 -91808
-42275 -85129
-38569 -78491
-35723 -86719
-45835 -86047
-42562 -92127
-48698 -89556
-35047 -97415
-36870 -79968
-46994 -89818
-33504 -77399
-42059 -99087
-50922 -86465
-40797 -77164
-41699 -79740
-43211 -96900
-48454 -85409
-48608 -90598
-45734 -93583
-40469 -84374
-33701 -97855
-47772 -86778
-30948 -99358
-40387 -87183
-37415 -92667
-32387 -97310
-34238 -92496
-41519 -98244
-49074 -92225
-33394 -97457
-45894 -92526
-32293 -82988
-30418 -87209
-31939 -87940
-37198 -78856
-36842 -80650
-40198 -75796
-42531 -95184
-48203 -90399
-38984 -83467
-43696 -76483
-50109 -87350
-50328 -79487
-48830 -89240
-48321 -78201
-45898 -87778
-41319 -97389
-46651 -87154
-31250 -84117
-34241 -79202
-46797 -81282
-46100 -81629
-41150 -76727
-38466 -82969
-34695 -97319
-35999 -97313
-47609 -84598
-41042 -89752
-32880 -96889
-31622 -97489
-38037 -93562
-50514 -86421
-37981 -84665
-34142 -82263
-41494 -96223
-50237 -76675
-34907 -82061
-47306 -81437
-45981 -96747
-42399 -94427
-34166 -82185
-40032 -79452
-46032 -81544
-48234 -94302
-39968 -83187
-31035 -84602
-30889 -97534
-51206 -92213
-32597 -82838
-46220 -99628
-42175 -93374
-47405 -79771
-33404 -89551
-31878 -94366
-37783 -77712
-39223 -82527
-47656 -85213
-43709 -89422
-45436 -75756
-44335 -80342
-37560 -96316
-32247 -94090
-35982 -89214
-34601 -99218
-49214 -90921
-48860 -86861
-44019 -97004
-38148 -81223
-30676 -78430
-42874 -97826
-47427 -90130
-46114 -87759
-50415 -77354
-48387 -87620
-48170 -83741
-30913 -80603
-36142 -89283
-42962 -93318
-44527 -76353
-43171 -89998
-33571 -82868
-49802 -94988
-37757 -86946
-43481 -86053
-33240 -86788
-44100 -82694
-45867 -91195
-47167 -90266
-35313 -76980
-32402 -79923
-42657 -91388
-49279 -89841
-39618 -95803
-45241 -96990
-46305 -86975
-38185 -94271
-38314 -90925
-40495 -94545
-35122 -82200
-37071 -88479
-34600 -87074
-45535 -92462
-44497 -89206
-31203 -86828
-41088 -99044
-33640 -92765
-36536 -79219
-45995 -99212
-40584 -87474
-46399 -86944
-40204 -95525
-32814 -88021
-38236 -100111
-40451 -88356
-49263 -79040
-43350 -81669
-50225 -98312
-37204 -95163
-38001 -79581
-34780 -89978
-33118 -80568
-49130 -86680
-40084 -99336
-47912 -92678
-35753 -88996
-32561 -76422
-43936 -98536
-32366 -78395
-38484 -98514
-36078 -93227
-49010 -77373
-49455 -92221
-38829 -76138
-30815 -80222
-44164 -76773
-34220 -84737
-49241 -75695
-38863 -98770
-35703 -91841
-34916 -82437
-50951 -96029
-47418 -80724
-39638 -78545
-37391 -84779
-35754 -98487
-31282 -88993
-33190 -88075
-38429 -87255
-40776 -81675
-48910 -88509
-45715 -80329
-34625 -95133
-42462 -96134
-41987 -91750
-42886 -83158
-37629 -99953
-40386 -93521
-38399 -80319
-36925 -78642
-47485 -99436
-32660 -86289
-32803 -79314
-41115 -96612
-39414 -76360
-41363 -94017
-46922 -84684
-43090 -90531
-41057 -77843
-33871 -97996
-44187 -92645
-33020 -83786
-36079 -94994
-37771 -82874
-51337 -82842
-39731 -84403
-45573 -94058
-44970 -98972
-49864 -80339
-39838 -91651
-38994 -92775
-50447 -98838
-38750 -95902
-30514 -90203
-33710 -95060
-31081 -93242
-32526 -75951
-34679 -81903
-32130 -77873
-35019 -98301
-30995 -81228
-42498 -79615
-35350 -94168
-36589 -81379
-40630 -78225
-46368 -89016
-36003 -90093
-31549 -91453
-33668 -78242
-35182 -89615
-35294 -77864
-32632 -93340
-40551 -98787
-38931 -88855
-43165 -75895
-47842 -76338
-43470 -88564
-48392 -75858
-44313 -80085
-41304 -91325
-40802 -82862
-48142 -91023
-33690 -76145
-44729 -89159
-50858 -89637
-31711 -76338
-48315 -94827
-31800 -92345
-31452 -88721
-41902 -79704
-31969 -89197
-39498 -82675
-33884 -91163
-31911 -77400
-44636 -95395
-36103 -98471
-41713 -83107
-46298 -84122
-30911 -77496
-32435 -97194
-47965 -82952
-36695 -94760
-35427 -80029
-46478 -87724
-37642 -76464
-46186 -77310
-41386 -92607
-41121 -94634
-46645 -80292
-36911 -86897
-40317 -86278
-48985 -84390
-38476 -87288
-45710 -80012
-35397 -89783
-45416 -76185
-31982 -81002
-48663 -98860
-33112 -77011
-32603 -100279
-46226 -96870
-37901 -81492
-47488 -85189
-46529 -91679
-36598 -84666
-42272 -100018
-37839 -89033
-33834 -76155
-34288 -82629
-47361 -99209
-41637 -76390
-39597 -84813
-50505 -83640
-49918 -85617
-31451 -76491
-48291 -90424
-31047 -89289
-40558 -91078
-51225 -86101
-41265 -98111
-43509 -94775
-46178 -76754
-50092 -77718
-39823 -95702
-32530 -81873
-41550 -96204
-49094 -91623
-51257 -83333
-39433 -99246
-41958 -77850
-39576 -98312
-45488 -86147
-41768 -79694
-36986 -93468
-30566 -84280
-39997 -80593
-32497 -97636
-47768 -93109
-40057 -85201
-37021 -91325
-37718 -76174
-35014 -85676
-43222 -80098
-47206 -79566
-50040 -90764
-47569 -86131
-46169 -82760
-43156 -81334
-38382 -78535
-32105 -79540
-49034 -90045
-44492 -96892
-45770 -90173
-32718 -79462
-42693 -96400
-48174 -96694
-47403 -76428
-45114 -97967
-41389 -80850
-40048 -77829
-34910 -92873
-32940 -85250
-35773 -76440
-50952 -95537
-39526 -99245
-39922 -75797
-44574 -82040
-48356 -77955
-39145 -81016
-50642 -94287
-39659 -99122
-32254 -85624
-45328 -89077
-44511 -94499
-43097 -78324
-35961 -95735
-35795 -93584
-50525 -97898
-31087 -93273
-31880 -86643
-44196 -94815
-36697 -78514
-33473 -77794
-49236 -84668
-38478 -96719
-41580 -92891
-36384 -75786
-31644 -77493
-36169 -80586
-51033 -80443
-33234 -93074
-46750 -100181
-31360 -93655
-30402 -84142
-49812 -82829
-48923 -99328
-49682 -95859
-42512 -88703
-34251 -83277
-45552 -93460
-48198 -76260
-30658 -83724
-36165 -92412
-39874 -85121
-47958 -77154
-31299 -81709
-44267 -93172
-31654 -96096
-33227 -77953
-30678 -87620
-37177 -97118
-36962 -99766
-43799 -96776
-31921 -82625
-46378 -76068
-35032 -89579
-47982 -95396
-49425 -81212
-31038 -87507
-33055 -97008
-51095 -88084
-40331 -86660
-39288 -99388
-32443 -91065
-31245 -79185
-38086 -84822
-36702 -100252
-30771 -96702
-41233 -79703
-39368 -91526
-40833 -78150
-37853 -90883
-50869 -86563
-39406 -94168
-32329 -94510
-30912 -96379
-49792 -86195
-49768 -80984
-39444 -77727
-35925 -99947
-35226 -96915
-31438 -94808
-36434 -97176
-43949 -90542
-41529 -83024
-31162 -98659
-36607 -76094
-46594 -75587
-35361 -77917
-43832 -82273
-31040 -96469
-30538 -95101
-48229 -79340
-35305 -97240
-47532 -78360
-38673 -92164
-46849 -88959
-35645 -78138
-43008 -84541
-37895 -78220
-48796 -98748
-44512 -88226
-37133 -89783
-46621 -87496
-45202 -94182
-40866 -99764
-35399 -79844
-41729 -87970
-30434 -79410
-38269 -81274
-36186 -81853
-36115 -76737
-40567 -84371
-38014 -86958
-38825 -100041
-32550 -88584
-37232 -87110
-42009 -90496
-40909 -90451
-43510 -82975
-38239 -85736
-46609 -84977
-48313 -80892
-40552 -95778
-48248 -97518
-49894 -83349
-40682 -94079
-47979 -87215
-32386 -94869
-30474 -80007
-36963 -96659
-31974 -85867
-42205 -91553
-44123 -87631
-50208 -92334
-42931 -79091
-31413 -95779
-35726 -97259
-37075 -90826
-49733 -99831
-35919 -94385
-34109 -84763
-43824 -77982
-50707 -100261
-40055 -81155
-38498 -97226
-33193 -88698
-47740 -75913
-48380 -94538
-39933 -99923
-31482 -77881
-41055 -79761
-37172 -78385
-37809 -80609
-46508 -93672
-38451 -95022
-31440 -92220
-44554 -88323
-34319 -85750
-36736 -99391
-45257 -92065
-33471 -83893
-34779 -95280
-39095 -77952
-48452 -90602
-40878 -89092
-38788 -80447
-38774 -97031
-39761 -76561
-30587 -92471
-32141 -92575
-40537 -87856
-43031 -78152
-42010 -77156
-46039 -93216
-36803 -91922
-42694 -92781
-31753 -79449
-50658 -98565
-38417 -83402
-45341 -79455
-38309 -100027
-37066 -94829
-48492 -86901
-39021 -83429
-39219 -79816
-39716 -96772
-33892 -88791
-44189 -82848
-48864 -78532
-43175 -96873
-31857 -82406
-40026 -82831
-42253 -93796
-30473 -92295
-32199 -78280
-35087 -91979
-42170 -86685
-49398 -99738
-32476 -75867
-45514 -77173
-41470 -77156
-39560 -79659
-31318 -76756
-38540 -79298
-48092 -79244
-49923 -96876
-37988 -84839
-49266 -79170
-49684 -92691
-47307 -83979
-46444 -98709
-47758 -81614
-47334 -90960
-40951 -85504
-40526 -98651
-32212 -79127
-49898 -100258
-46954 -85380
-48992 -87875
-51006 -99150
-40947 -75753
-49091 -76841
-46536 -92873
-37163 -93583
-51031 -81631
-47794 -92427
-34352 -88776
-37847 -99459
-41906 -78400
-45811 -77087
-35816 -85104
-31645 -77280
-47612 -91357
-43704 -89329
-37804 -91778
-42097 -91142
-39048 -79169
-50351 -94949
-49853 -89581
-36257 -89584
-47678 -95324
-37113 -80885
-48593 -98877
-43727 -76466
-43215 -100256
-31373 -84234
-35698 -98036
-45546 -79631
-41277 -78193
-37832 -80107
-31217 -81755
-33152 -79883
-41795 -98533
-38758 -80638
-35991 -98054
-31109 -80806
-37355 -82514
-36557 -79288
-37173 -99191
-49635 -83609
-48368 -84077
-39878 -77331
-35263 -82282
-43806 -85893
-31426 -80411
-48489 -76091
-30880 -86428
-50250 -99879
-39059 -98058
-41707 -79319
-36174 -82939
-30724 -86237
-43147 -85484
-47166 -75584
-41524 -78072
-46270 -91674
-41035 -95496
-46008 -97150
-49793 -77686
-31049 -96729
-39689 -80175
-47807 -81651
-33699 -94152
-45845 -78840
-49450 -79274
-47431 -93928
-50130 -92187
-38857 -76161
-36352 -77453
-50397 -83908
-33481 -75696
-32941 -89728
-43793 -75519
-41908 -85797
58614 14361
69615 4501
77972 373
62276 2914
60958 7332
72468 -7676
63426 -412
73739 1749
64571 9089
64303 -8130
73531 4644
65284 6052
59109 5105
74094 14911
70038 6355
63602 15520
69595 -6030
62071 12464
68910 7181
67163 -3510
73175 -6855
76743 12053
62043 -6871
75711 -6146
72897 4810
62798 2998
69365 -5188
69672 8942
61777 4073
62866 -1144
75951 170
67507 -4947
63510 5507
60963 12082
76208 14604
73818 2197
66768 8509
70602 860
69547 9894
77243 13251
60526 10702
74876 4796
60805 6228
77230 -4954
56850 -3499
67793 8726
76860 2825
55657 6609
70439 -992
63069 -3361
73266 3236
56745 8748
59710 207
74533 10180
57733 14800
58946 -1513
65196 -7795
75126 -3346
58488 16022
58083 446
65787 -5430
75057 -8135
60426 -2063
56391 9144
62826 9282
67706 -4664
71604 -8206
76637 -7912
64790 805
73719 -5680
76521 -2510
70557 8866
61612 -4529
75939 3048
61941 11006
69009 13843
78476 -7213
58333 3601
56219 4526
64206 14771
68496 8100
69969 -2770
64779 5277
71462 10342
64412 -5703
56791 15596
65249 -965
62316 15501
55941 14413
75521 14014
61956 1333
58290 11405
76615 4784
61251 -7791
58615 6592
75109 14428
71648 15999
63186 9326
77093 16095
59536 16014
74248 2822
58099 13966
66538 11372
73406 -7014
75254 12286
69193 -3267
72974 6956
57132 -3015
64296 1643
63130 1039
62993 12963
72702 6001
72492 1768
60364 13706
70283 14271
57428 8184
59229 7673
68356 8461
72738 8643
74381 7378
72813 13707
60923 8895
73335 13236
69089 -3900
56206 -2404
67384 5851
77161 -6242
69087 14948
71743 3134
55861 -4534
67402 7504
76766 15073
76790 6842
71532 -7144
59798 11195
59368 12229
59269 -5059
72827 233
58238 9563
71709 16021
68616 7810
57947 -2553
69762 489
67787 7866
55735 3128
75837 10455
65389 1455
77857 872
61775 7545
55741 7989
73595 2652
66411 10138
72599 15047
59600 11174
65148 2850
78753 11481
63509 12401
71224 4070
66510 -5758
55889 -1267
62354 -2427
76738 15771
69925 10489
63942 66
63729 10130
66893 14346
74235 -4406
75479 8257
59155 12915
76591 -240
62183 16220
63055 7407
71309 13247
62366 -8242
66750 -1292
72535 7784
62256 -6038
67540 13926
65962 -5021
62027 -3609
73976 -4211
60915 -7710
73928 7542
62827 -4537
71087 11567
76352 -1933
58013 -3926
62456 -1396
63129 5032
76874 10440
57092 -6897
60061 2248
57189 2582
56733 7299
61992 12686
55874 -2034
73617 15752
61516 12477
58991 -1957
71979 10605
60265 15797
64651 -6387
57784 15047
60269 -549
64087 -4533
75746 -7484
59786 3916
76098 3704
59138 10194
56701 1285
55590 11971
62208 114
77477 -6505
77750 -1884
63391 1860
59180 5976
70917 13117
65395 -469
63245 16239
71099 9765
71510 4399
72560 -2027
57717 13662
63719 6576
67966 -5335
70860 3497
56295 10996
65229 3981
57997 11112
63583 13592
55785 12909
60474 6316
77822 9872
66945 -1426
65601 6191
73949 -1057
57462 7744
58844 -5349
72375 8829
69271 6067
76802 -908
62144 12024
57478 8648
68024 -129
57624 7309
68781 13716
61315 11673
78148 7840
65507 -7192
58643 15217
78256 10288
78192 14458
70995 12234
67925 495
64149 16159
73273 13085
68218 -1578
59263 -3781
62046 646
69720 1749
64191 8351
63997 -7754
56030 1339
59676 9548
64567 6906
76851 14811
63290 10982
65759 11220
56343 10529
67754 -1958
68087 6971
59116 472
56032 -7261
74045 -5275
75599 11933
68480 12977
75954 2359
61034 10900
71761 3700
72482 15157
78651 15683
62926 -2245
69844 6868
71286 15261
68269 418
57200 9720
73676 12908
58159 10996
58993 6586
77088 5583
70325 -4780
56423 -673
65902 -5287
72159 2267
66629 14079
60351 -3031
63726 14744
78280 6586
66123 7859
75611 15662
77872 10557
69227 14234
78007 -7150
68184 15813
65499 -5801
58649 5291
67047 -1306
71221 6019
72367 11750
68472 -4091
63859 3664
60081 -3437
78562 -6433
60428 357
72504 8634
65653 5253
59750 -5987
78355 10854
58871 -4945
59879 13441
73526 -2750
69031 15789
64161 5891
76305 15617
61743 6802
69701 10748
62762 12328
69011 14883
58951 -1425
59121 -299
77717 -3464
76958 -1746
61823 2858
58041 8868
60093 -7489
65116 -2705
67293 -5657
68318 -150
57110 4844
76419 8773
57037 -4240
60368 -7296
62178 -7182
61139 11523
59834 -1555
67077 10892
70189 -5432
69075 11270
71461 16419
71614 7736
66792 7449
65986 14670
73413 -4993
71860 -1480
63547 -8239
56414 -2316
71012 4675
61522 -5807
74170 3204
75097 8070
60887 5280
57388 16123
59990 5406
68072 -5775
76909 -752
76685 15323
75015 8869
66350 -493
70009 15180
59978 -3973
57754 12953
73489 15546
65154 -4634
58090 -5641
74730 7220
60635 1887
56095 -5556
75548 380
77334 -7692
63159 -2734
76181 -7909
67067 14768
76257 -363
65898 -1818
66087 -2337
62110 -3124
61693 12346
61797 10348
69144 3101
55883 1584
60584 9659
62899 4830
63791 -5109
66155 -747
66105 -6140
61965 -8256
56321 9138
75766 -570
57650 1265
56275 -3959
60961 817
67802 -6571
71209 10368
75891 11387
58935 -1639
73400 12463
78527 467
76321 9814
63516 -4562
69855 3539
69337 -897
63843 -6330
61841 14129
76364 15303
68248 9538
67096 -5824
65234 -1804
66379 15794
72695 4869
78438 -8111
57453 15205
78727 -190
74878 12673
78572 13964
67166 424
66127 11682
76869 12616
71805 1571
57940 13020
73025 12300
60663 -5874
70315 107
75372 8883
63929 8556
74160 3634
68638 -216
65053 1778
55863 -6507
75853 -5977
63409 14192
67607 -111
68457 9459
55978 14236
72758 5799
70312 13612
62373 13189
77624 -2866
66512 -2657
59534 15433
65039 -1494
65708 14292
76643 -4185
64056 78
78424 14894
78118 381
70542 13138
62776 14843
71858 6238
76426 -155
66612 4889
60327 8428
64637 3248
59605 -1170
69278 9455
76925 5296
68518 -8054
76563 -6817
63591 13142
77599 11272
76102 10092
59251 11223
71807 15574
77814 -2574
69171 -2413
57704 -6529
61216 -6106
77238 15539
72832 6478
78623 9731
62232 -6562
64169 9501
68189 6564
75607 553
71857 1449
69698 13551
59503 13142
66868 7770
58338 2038
59973 3702
72915 3742
70050 -6766
69955 -1888
78627 15535
76873 -6160
59506 -1335
56684 10367
77498 10679
75785 -7453
65591 -3959
74194 8707
63355 4192
72270 -1198
75661 -779
77873 1434
75571 9885
71116 8782
69885 -8181
60107 -1447
55557 443
55717 -386
72413 13663
58613 7896
68740 5173
55617 -5752
69082 -1649
71607 14281
61220 -2038
71624 -3632
71509 10601
56787 238
76573 12950
73570 13241
68224 4874
77077 1977
68322 14780
59271 711
75996 14694
65942 -5295
60276 -7956
69743 11588
72787 15453
65669 4797
72863 -2789
69282 -2974
56854 -1229
56629 373
66780 672
58320 11220
64836 4544
65494 -3846
73133 -3148
65741 -3663
74490 7702
74126 8245
71085 9532
76449 -5866
70086 10533
72921 9019
60844 -1737
64978 -7113
72862 16458
60362 -2875
62277 -3105
78373 -7109
70310 14059
57500 3975
58237 3143
59184 13631
71541 14216
59590 10396
67868 7115
64272 -2430
64625 12665
76498 -5174
63032 16482
65353 1865
76326 986
57805 11251
69136 -1147
68408 3975
74959 -5940
69793 -4319
62885 -7489
63490 11621
71022 -3969
78428 12284
62843 -2201
67729 -1050
60702 3415
68125 2719
75903 10530
72030 15069
56367 14705
77305 10798
57993 -2215
69797 478
78690 3856
73134 -3468
77842 -4299
75856 6359
73320 -3515
76997 11785
76478 16271
58010 -1745
57772 -7562
72114 15600
78238 14783
65299 13403
68287 -7294
70766 5833
69196 -2504
75100 13011
59568 -324
61160 -669
73213 -4530
63202 6783
57735 8174
64150 2816
67061 13495
57361 1828
68016 14724
56197 -7582
66897 14296
63231 -926
75666 -4244
78723 -4549
65650 8347
56774 8622
63374 10605
59819 2624
55523 13519
56403 6285
63011 16209
55752 15465
75703 2942
77754 7434
65938 3301
78515 10684
59063 8164
70297 5941
60289 7916
61430 5070
61098 3843
55867 -297
71586 13728
58101 -4052
60538 -2194
75792 10293
57694 15517
58863 -655
55725 -6026
72875 7519
59794 12661
70889 11917
61751 9962
66423 -1079
63163 -6473
76625 11402
77726 -5889
74014 1843
75979 5873
62750 12533
61845 3322
72236 7670
78606 9806
68376 11242
58665 -5602
78009 15002
74841 15930
58039 6320
65564 6157
63033 -7126
78634 -319
73192 8893
74052 10684
67768 -5985
64938 15099
75747 -1037
63938 12279
55556 16433
63485 9400
64950 -1371
66564 -6630
60748 5767
64505 13315
78762 -8218
74233 3168
69705 7445
73675 975
74018 2119
71236 5788
56540 -1408
62577 968
70834 735
62829 10538
62568 10771
61942 11704
76123 15045
60959 693
70131 13113
62015 -1140
61198 12159
60393 8018
57413 5369
72462 9084
55777 9649
75312 4717
63598 11394
60012 -6922
68579 -7439
68798 6797
71394 -4874
62782 -844
64304 3543
62807 11073
61896 15774
68580 7425
72750 13243
60123 -1228
77593 -5649
73088 -1703
71168 -4310
63915 5876
64915 14573
74399 -4591
62216 -4933
62017 47
59941 13438
69123 -7477
74025 2681
57414 1951
74969 3767
70081 3845
67225 5485
64967 -3462
67016 1171
56411 9733
72948 310
74382 2078
62339 9904
77927 7518
77733 -7812
62799 -71
75991 9406
66192 12142
73224 8163
61665 4338
67274 -6559
61190 7761
74660 12681
61966 -6010
59509 13572
71995 6642
67485 2638
56132 7839
75099 -2576
62472 13156
71740 13850
65314 4277
66580 1071
66284 16462
78670 12196
74208 -4130
74535 8958
60004 13303
56176 12938
57583 12158
77211 9872
68790 -7220
77883 -1982
60416 6894
67495 -3615
58799 3351
62279 3452
55533 -6778
73120 3619
62639 2496
58676 2916
63331 5522
60127 12888
65042 -1318
69304 10648
66466 -2491
65553 14154
57498 11964
70522 8473
78222 -547
60658 12763
75466 -629
56066 35
74962 -3702
58355 -7081
70386 4618
60164 15041
58534 15315
76752 15281
67995 7508
65184 9345
76772 1808
69391 14729
77207 15194
75781 7613
58868 -6096
75976 -7335
65513 5930
69871 -6855
67017 3687
67551 908
69710 6179
62722 7565
69359 15854
56534 -2394
59930 14414
75354 -886
77560 4080
76780 8960
74789 -836
65447 11338
66480 -610
74059 12483
68195 -3875
74911 -6776
68157 337
56195 -5133
64635 -5145
66704 -2978
68675 9414
57474 12415
65416 9991
70939 6908
65570 15165
68670 15089
75127 -6697
68491 5530
59801 -2274
61108 -1617
76033 3210
56583 12522
73402 7922
77729 7388
77429 15629
72261 7480
71040 8457
55531 7752
68639 -4415
61153 13646
78350 9133
55716 -2462
63711 4620
58107 2574
70857 13119
71300 -5776
57910 2511
77786 16028
65451 -6593
67019 -1646
67391 5609
71927 -8148
66749 10345
76272 3879
69283 -2686
72053 13044
71428 13777
70539 15310
57265 14925
57868 1591
63420 239
60622 12486
61223 -5291
74785 2935
61234 2329
68151 11757
75706 8443
72104 -8054
72877 -4761
57778 -6436
65660 -6387
66638 -1401
70016 5123
76467 3894
61957 2175
56140 -5465
74646 13205
60417 3868
72878 15896
72108 1731
60356 -363
76317 8392
63848 -586
65547 -5708
55535 3476
61078 2455
55603 -6763
76275 -5762
70541 11680
58119 -6389
75320 -2284
73006 6671
56378 588
75221 -1425
77600 6081
76708 12188
58689 -1790
78033 2848
59890 2221
78051 14382
65620 -2053
58574 -5245
60792 8672
59462 1946
75833 15208
58737 -6192
73771 1586
67609 15405
