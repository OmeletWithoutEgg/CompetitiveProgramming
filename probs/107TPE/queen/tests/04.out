11054.80 -43536.03
